database/queries/like_career.js

```diff
@@ -4,7 +4,8 @@ const likeCareer = (data, callback) => {
   const likeCareerQuery = `INSERT INTO users_careers(user_id, career_id) VALUES($1, $2) RETURNING (user_id, career_id)`;
   dbConnection.query(likeCareerQuery, data, (error, response) => {
     if (error) {
-      callback(error);
+      console.log(error);
```

Comment:
Log in this query but not the others. I don't mind logs for server errors but keep it consistent.